### PR TITLE
Apply new oauth2 rate limit buckets

### PR DESCRIPTION
**Pull request by peteryaworsky-okta**
Resolves: OKTA-217213

@larsjohansen-okta 
@prasanthmathialagan-okta 
image
image

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 1]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
 | **Get System Log data:**<br>`/api/v1/events`                                                                 | 20               | 25               | 25      | 50         | 1000               |
 | **Get session information:**<br>`/api/v1/sessions`                                                           | 100              | 300              | 300*    | 600*       | 750                |
 | **Create an organization:**<br>`/api/v1/orgs`                                                                | N/A              | N/A              | N/A     | 50         | 50                 |
-| **Authorize request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/authorize`             | 100              | 300              | 300*    | 600*       | 10000              |
-| **Token request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/token`                     | 100              | 300              | 300*    | 600*       | 10000              |
+| **All actions on a custom oauth2 Authorization Server :**<br>`/oauth2/{authServerId}/v1`                     | 300              | 600              | 600*    | 1200*      | 2000               |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Suggestion: `All actions on a Custom Authorization Server`

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 2]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
...
+| **All actions on a custom oauth2 Authorization Server:**<br>`/oauth2/v1`                                     | 300              | 600              | 600*    | 1200*      | 2000               |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Suggestion: `All OAuth2 requests for an Org Authorization Server`

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 3]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
...
+| **Create or list oauth2 clients:**<br>`/oauth2/v1/clients`                                                   | 25               | 50               | 50*     | 100*       | 100                |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Suggestion: `All OAuth2 client configuration requests`

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 4]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
...
+| **All other oauth2 actions:**<br>`/oauth2`                                                                   | 100              | 300              | 300     | 600        | 600                |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Suggestion: `All other OAuth2 requests`

**Comment by peteryaworsky-okta** [comment]
@larsjohansen-okta I made the changes you suggested. Do you think it looks better if we remove the word "All" from these 4 lines except the "All other OAuth2 requests" one?

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 5]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
 | **Get System Log data:**<br>`/api/v1/events`                                                                 | 20               | 25               | 25      | 50         | 1000               |
 | **Get session information:**<br>`/api/v1/sessions`                                                           | 100              | 300              | 300*    | 600*       | 750                |
 | **Create an organization:**<br>`/api/v1/orgs`                                                                | N/A              | N/A              | N/A     | 50         | 50                 |
-| **Authorize request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/authorize`             | 100              | 300              | 300*    | 600*       | 10000              |
-| **Token request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/token`                     | 100              | 300              | 300*    | 600*       | 10000              |
+| **OAuth2 requests for a Custom Authorization Server:**<br>`/oauth2/{authorizationServerId}/v1`               | 300              | 600              | 600*    | 1200*      | 2000               |
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: It is for all custom AS, correct? If yes, then it should be `OAuth2 requests for Custom Authorization Servers`.

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md [hunk 6]
```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
...
+| **OAuth2 requests for an Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients`             | 300              | 600              | 600*    | 1200*      | 2000               |
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: There is only one Org AS, so `OAuth2 requests for the Org Authorization Server`

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md (hunk 5 above)
Comment (peteryaworsky-okta) [reply packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Good point – will change.

File: packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md (hunk 6 above)
Comment (peteryaworsky-okta) [reply packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md]: Good point – will change.

**Pull request closed without merging**